### PR TITLE
Add/pull request template

### LINKED ISSUE #14: [Documentation] Add PR Template
Description:
The PR template for a consistent, efficient PR process is missing. 
 
Expected Behavior:
Ensure contributors follow a standard process for every pull request and will have a to-do list to check before requesting a review. Can you please assign it to me @A91y ? Thanks

### PR DESCRIPTION
**Pull request by LaraKraemer**
### Describe your changes
I have added a PR template in a designated .github folder. I also moved the code of conduct file into a .github folder. These two steps follow GitHub open-source best practices. 

### Issue ticket number and link
fixes #14 

### Checklist before requesting a review
* [x] I have performed a self-review of my code
* [x] If it is a core feature, I have added thorough tests.
* [x] I have followed the guidelines in the Contributing document.
* [x] I have added / updated documents (for bug fixes / features).

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+
+### Issue ticket number and link
+Fixes #relatedIssueNumber
+
+
+### Checklist before requesting a review
+* [] I have performed a self-review of my code
+* [] If it is a core feature, I have added thorough tests.
+* [] I have followed the guidelines in the Contributing document.
```
Comment (A91y) [inline .github/pull_request_template.md:11]: Please link the Contributing.md page here for direct reference.

**Review by A91y — CHANGES_REQUESTED** [review]

**Comment by LaraKraemer** [comment]
Added the reference link

**Review by A91y — APPROVED** [review]
lgtm

**Comment by A91y** [comment]
Thanks @LaraKraemer

**Pull request merged**